infra/fastly/terraform/dl.k8s.io/vcl/binaries.vcl

```diff
@@ -109,6 +109,16 @@ sub vcl_hit {
 
 sub vcl_deliver {
 
+  set resp.http.Content-Security-Policy = "default-src 'self'";
+  set resp.http.X-Frame-Options = "SAMEORIGIN";
+  set resp.http.X-XSS-Protection = "1";
+  set resp.http.X-Content-Type-Options = "nosniff";
+  set resp.http.Referrer-Policy = "origin-when-cross-origin";
+
+  if (req.protocol == "https") {
+    set resp.http.Strict-Transport-Security = "max-age=63072000; includeSubDomains";
```

Comment:
Could add a comment about how we picked the max age.

Reply:
@sftim Done. PTAL